essence-of-live-coding/src/LiveCoding/Migrate/NoMigration.hs

```diff
@@ -0,0 +1,92 @@
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE Arrows #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE DeriveFunctor #-}
+{-# LANGUAGE DeriveTraversable #-}
+
+{-|
+Module      : LiveCoding.Migrate.NoMigration
+Description : Mechanism to save state in a Cell without requiring a Data instance.
+
+If a data type is wrapped in 'NoMigration' then it can be used in a 'Cell' without requiring 
+it to have a 'Data' instance. The consequence is that if the type has changed
+in between a livereload, then the previous saved value will be discarded, and no migration attempt
+will happen.
+
+'LiveCoding' does not export 'delay' and 'changes' from this module. These functions should be
+used with a qualified import.
+-}
+module LiveCoding.Migrate.NoMigration where
+
+-- base
+import Data.Data
+    ( Data(dataTypeOf, gunfold, toConstr),
+      Typeable,
+      mkConstr,
+      mkDataType,
+      Constr,
+      DataType,
+      Fixity(Prefix) )
+
+-- essence-of-live-coding
+import LiveCoding.Cell ( Cell(Cell) )
+import LiveCoding.Cell.Monad ( hoistCellKleisli )
+import Control.Arrow ( Arrow(arr), returnA, (>>>) )
+import qualified LiveCoding.Cell.Feedback as Feedback
+import Control.Monad (guard)
+
+-- |Isomorphic to @'Maybe' a@ but has a different Data instance
```

Comment:
```suggestion
-- | Isomorphic to @'Maybe' a@ but has a different 'Data' instance.
```
Maybe some description what it's used for? Or is the module supposed to be internal?

Reply:
Will add description. The module is not internal.